server/protocol/join.js

```diff
@@ -59,7 +68,22 @@ function validateJoinerAddress(ringpop, joiner, callback) {
         return false;
     }
 
+    var blacklist = ringpop.config.get('memberBlacklist');
+    if (Array.isArray(blacklist) && anyBlacklisted()) {
+        callback(BlacklistedError({
+            joiner: joiner,
+            blacklist: blacklist
+        }));
+        return false;
+    }
+
     return true;
+
+    function anyBlacklisted() {
+        return _.any(blacklist, function any(pattern) {
```

Comment:
Since you already verified this is an Array, why not just use `Array.prototype.some`? If you add an `isArray` check to the validator, you could also use `Array.prototype.every` there